dreamhome.py

```diff
@@ -0,0 +1,9 @@
+from flask import Flask
```

Comment:
Great initial runner - we may want to call this file `app.py` (convention, not an error)